packages/amplify-cli/src/extensions/amplify-helpers/update-amplify-meta.ts

```diff
@@ -6,6 +7,7 @@ import {
 } from 'amplify-cli-core';
 import { BuildType } from 'amplify-function-plugin-interface';
 import * as fs from 'fs-extra';
+import { fs as vfs } from 'memfs';
```

Comment:
does this work on windows reliably ?

Reply:
Yes, it works for windows. I tested it yesterday in a windows machine. I will elaborate more in the PR description today